webhook_to_fedora_messaging/config/config.toml

```diff
@@ -0,0 +1,60 @@
+[flaskapp]
+DEBUG = true
+TESTING = true
+PROPAGATE_EXCEPTIONS = ""
+SECRET_KEY = "PLEASE-CHANGE-THIS-BEFORE-STARTING"
+PERMANENT_SESSION_LIFETIME = 604800
+USE_X_SENDFILE = false
+SERVER_NAME = ""
+APPLICATION_ROOT = "/"
+SESSION_COOKIE_NAME = "user"
+SESSION_COOKIE_DOMAIN = ""
+SESSION_COOKIE_PATH = ""
+SESSION_COOKIE_HTTPONLY = true
+SESSION_COOKIE_SECURE = false
+SESSION_COOKIE_SAMESITE = ""
+SESSION_REFRESH_EACH_REQUEST = true
+MAX_CONTENT_LENGTH = ""
+SEND_FILE_MAX_AGE_DEFAULT = ""
+TRAP_BAD_REQUEST_ERRORS = ""
+TRAP_HTTP_EXCEPTIONS = false
+EXPLAIN_TEMPLATE_LOADING = false
+PREFERRED_URL_SCHEME = "http"
+TEMPLATES_AUTO_RELOAD = ""
+MAX_COOKIE_SIZE = ""
+
+  [flaskapp.database]
```

Comment:
This will not be used anymore, `sqlalchemy-helpers` uses a config key named `SQLALCHEMY_DATABASE_URI` (like `flask-sqlalchemy`)

Reply:
Gotcha. Nuking the entire database section of the example config.